src/core/FFMPEGMovie.cpp

```diff
@@ -445,28 +444,19 @@ bool FFMPEGMovie::convertVideoFrame()
 
 bool FFMPEGMovie::generateSeekingParameters()
 {
-    den2_ = videoStream_->time_base.den * videoStream_->r_frame_rate.den;
-    num2_ = videoStream_->time_base.num * videoStream_->r_frame_rate.num;
-
-    if( !( den2_ > 0 && num2_ > 0 ))
-    {
-        put_flog( LOG_WARN, "cannot determine seeking paramters, file not supported." );
-        return false;
-    }
-
-    numFrames_ = ( videoStream_->duration > 0 ) ?
-                 av_rescale( videoStream_->duration, num2_, den2_ ) : 0;
-
-    frameDurationInSeconds_ = (double)videoStream_->r_frame_rate.den /
-                              (double)videoStream_->r_frame_rate.num;
-
-    put_flog(LOG_DEBUG, "seeking parameters: start_time = %i, duration_ = %i, numFrames_ = %i",
-             videoStream_->start_time, videoStream_->duration, numFrames_);
-    put_flog(LOG_DEBUG, "                    frame_rate = %f, time_base = %f",
-             (float)videoStream_->r_frame_rate.num/(float)videoStream_->r_frame_rate.den,
-             (float)videoStream_->time_base.num/(float)videoStream_->time_base.den);
-    put_flog(LOG_DEBUG, "                    frameDurationInSeconds_ = %f",
-             (float)frameDurationInSeconds_);
+    numFrames_ = videoStream_->nb_frames;
```

Comment:
This looks simpler, but are you sure that videoStream_->nb_frames is always set and valid?